### ISSUE TITLE
Creating a weekly editorial digest doc

### ISSUE THREAD

**Issue by rubymorillo** [hoodiehq/editorial#82]
Hey team,

**The Problem:** Some of the issues in the Editorial repo have "in progress" tags while others don't; some issues have long convos in them which are hard to follow and don't make it clear who has claimed an issue and what the status of it was (ongoing, blocked, paused, etc). Additionally, it's not clear what will be in the pipeline over the coming weeks which would allow current and future contributors to be able to plan accordingly in case all current issues already have people working on them.

**A possible solution:** A weekly digest doc (one page) which we disseminate after the Wednesday editorial standup (or sometime on Wednesday) which list out the following:
- Claimed issues and their statuses (if any are paused or if a contributor dropped off, this would be a good way of letting other folks who have bandwidth work on them. This would also mean asking contributors to offer a weekly update in their issues so everyone knows where we are).
- Old issues (it would be nice to prioritize issues that are > 3 months old so we can start closing those out).
- What's in the pipeline (it doesn't have to be exact, but if we know that over the next week or two we will have one or two blog posts in the works, or website updates, it would be great to let people know that these are coming versus checking in on the repo every so often).

The doc can then be added as a link in the #editorial channel's topic section to allow newer contributors to be brought up to speed on what's going on. 

Thanks!

**Comment by gr2m** [comment]
Great initiative, thanks @rubymorillo

**Comment by renrutnnej** [comment]
I'm late to chime in but I love this idea! My only concern is having someone who stays on top of this.
